src/pages/Counter/containers/CounterContainer.js

```diff
@@ -0,0 +1,54 @@
+import { Component } from "react";
+import CounterView from "../components/CounterView";
+import Layout from "../components/Layout";
+
+class CounterContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      countValue: 0,
+      isEven: true,
+    };
+  }
+
+  handleIncrement = () => {
+    this.setState((state) => {
+      const incrementValue = state.countValue + 1;
+      return {
+        countValue: incrementValue,
+        isEven: !this.state.isEven,
```

Comment:
you need to calculate is even or not every time. because if I add button +2 for example, the logic could possible work incorrect. Recalculate incremented value pls

Reply:
fixed the bug